Source/WebCore/platform/graphics/coretext/DrawGlyphsRecorderCoreText.cpp

```diff
@@ -379,6 +388,42 @@ void DrawGlyphsRecorder::recordDrawImage(CGRenderingStateRef, CGGStateRef gstate
     m_owner.translate(0, -(rect.size.height + 2 * rect.origin.y));
 }
 
+void DrawGlyphsRecorder::recordDrawPath(CGRenderingStateRef, CGGStateRef gstate, CGPathDrawingMode drawingMode, CGPathRef coreGraphicsPath)
+{
+    updateCTM(*CGGStateGetCTM(gstate));
+    auto gstateTransform = m_owner.getCTM();
+    // The path we get has already CTM applied to it but we should serailize the non-transformed version to correctly apply line width.
+    CGAffineTransform invertTransform = CGAffineTransformInvert(gstateTransform);
```

Comment:
`updateCTM()` sets the owner CTM to `CGGStateGetCTM()`. I think we can write this code like this:
```
auto* ctm = CGGStateGetCTM(gstate);
updateCTM(*ctm);
auto invertTransform = CGAffineTransformInvert(*ctm);
```